charts/opensearch/values.yaml

```diff
@@ -250,7 +250,7 @@ podManagementPolicy: "Parallel"
 # If you experience slow pod startups you probably want to set this to `false`.
 enableServiceLinks: true
 
-protocol: http
+protocol: https
```

Comment:
Why does the protocol here have to change?

Reply:
> I also wonder if you could squash some of the commits down - 54 commits seems pretty big!

We will squash and merge @smlx

Reply:
I changed the default value for some testing, can revert it back, no rationale